pkg/template/registry/templateinstance/strategy.go

```diff
@@ -52,8 +52,9 @@ func (templateInstanceStrategy) Canonicalize(obj runtime.Object) {
 func (templateInstanceStrategy) PrepareForCreate(ctx apirequest.Context, obj runtime.Object) {
 	templateInstance := obj.(*templateapi.TemplateInstance)
 
-	//TODO - when https://github.com/kubernetes-incubator/service-catalog/pull/939 sufficiently progresses, remove the if nil check and always
-	// Clear any user-specified info, then seed it from the context
+	// if request not set, pull from context; note: the requester can be set via the service catalog
+	// propagating the user information via the openservicebroker origination api header on
+	// calls to the TSB endpoints (i.e. the Provision call)
 	if templateInstance.Spec.Requester == nil {
```

Comment:
@liggitt @pmorie - if you all could, please re-review / take a quick peek at `PrepareForCreate` and see if you agree.  I tried to capture in the comment above what we think is different from "typical" `PrepareForCreate`.  Namely:
- the entry points like `Provision` into the TSB will provide the user.Info (as the service catalog propagates it in the orig identity header) and we set the user onto the `templateInstance.Spec.Requester` there
- so if it is set, we don't nil out
- if it is not set, we pull from the context

Ultimately, we test this behavior in various places, most notably our impersonation tests:  https://github.com/openshift/origin/blob/master/test/extended/templates/templateinstance_impersonation.go#L288

I believe prior agreement was obtained on all this, but after some discussions with @jim-minter it seemed prudent to double-check one more time.

thanks

@bparees fyi